htdocs/comm/propal.php

```diff
@@ -1996,8 +1996,8 @@
 	include DOL_DOCUMENT_ROOT . '/core/tpl/extrafields_view.tpl.php';
 
 	// Amount HT
-	print '<tr><td height="10">' . $langs->trans('AmountHT') . '</td>';
-	print '<td class="nowrap" colspan="2">' . price($object->total_ht, '', $langs, 0, - 1, - 1, $conf->currency) . '</td>';
+	print '<tr><td height="10" width="25%">' . $langs->trans('AmountHT') . '</td>';
+	print '<td class="nowrap" colspan="2">' . price($object->total_ht, '', $langs, 1, - 1, - 1, $conf->currency) . '</td>';
```

Comment:
There is no reason to round here. If the total_ht is not rounded with 5 digit, it means a bug was introduced simewhere else. Data must be rounded when inserted not when output.